### PR TITLE
Homebase requested the addition of id.pub kin.one kin.pub

### PR DESCRIPTION
**Pull request by OpieDelivers**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)
* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
  * [x] This request was _not_ submitted with the objective of working around other third-party limits
  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Homebase is a non-profit, open-source project, focused on safeguarding human rights online. Our mission is to empower individuals with the ability to establish their unique online identities, each anchored by its own domain name and certificate. 
Homebase is both completely decentralized and integrates seamlessly with existing Internet technologies, allowing users to host their own web pages, post blogs, engage in social networking, and use services like true peer-to-peer chat clients and personal image libraries. 

Our commitment to decentralized user control over personal data and online privacy is uncompromising. As part of this commitment, we employ zero-knowledge encryption. This means that user data is not just encrypted but is structured in such a way that only the user can decrypt and access their information - even Homebase cannot access this data.
We aim to pioneer a new era of a decentralized internet, one where knowledge, privacy, and genuine social connections are paramount. 

Organization website: [https://www.homebase.id/]( https://www.homebase.id/)

Reason for PSL Inclusion
====

Homebase is designed with an individual-centric approach, empowering users with privacy and control over their own data. While we encourage users to secure their own unique domain names, we also offer free subdomains (e.g., username.id.pub, firstname.lastname.kin.one, username.pub.one). This ensures that secure online identities remain accessible to those who may not afford or be able to manage their own domain names.

These identities are all distinct and are isolated from one another to ensure privacy and security. For this reason, in Homebase's model, certificates, cookies and any site data set on one “subdomain” (user identity) is not readable or accessible by another user identity on the same subdomain. Identities on the same subdomain might be hosted on the same or on different servers.

The inclusion of these domains in the Public Suffix List is critical for Homebase to maintain this high level of user privacy and data security. It would significantly enhance the reach and trustworthiness of our platform and align with our mission of providing a personal, secure, private space on the Internet.

The domains "id.pub", "kin.one", and "pub.one" are registered with a commitment to the long-term vision of the project, intending to extend the availability of secure online identities to as many individuals as possible. We hope over time that many ISPs will also begin to offer such subdomains to Homebase users that they host.

Homebase is built with the ability for users to freely move their hosted solution, even hosting themselves, which is a unique feature not commonly seen in conventional platforms. Being a true peer-to-peer system, Homebase significantly differs from typical internet platforms and requires distinct user privacy and security considerations.

The request is being made by an administrator, advocating for the rights and privacy of potential users of the Homebase platform. Administrator email: [info@homebase.id](mailto:info@homebase.id)

Number of users this request is being made to serve: Estimating 50,000 actual individual users per domain.

DNS Verification via dig
=======

```
dig +short TXT _psl.id.pub
"https://github.com/publicsuffix/list/pull/1768"

dig +short TXT _psl.kin.one
"https://github.com/publicsuffix/list/pull/1768"

dig +short TXT _psl.kin.pub
"https://github.com/publicsuffix/list/pull/1768"

```
Results of Syntax Checker (`make test`)
=========

All tests passed

### CODE REVIEW

**Comment by OpieDelivers** [comment]
Any update on this?

**Comment by simon-friedberger** [comment]
Hi Jason,

I think it would be best to sort this under "Homebase" if that's the name of the project and to provide a project email address so somebody can be reached if you ever leave the project.

You also haven't filled in anything under number of users that this request will serve.

You also don't seem have the required remaining two years of validity.

**Comment by Seifert69** [comment]
@simon-friedberger the email is info@homebase.id
Estimating 50,000 actual individual users per domain. 
We've been working on Homebase since 2020, not sure what the required two years of validity is?

**Comment by simon-friedberger** [comment]
> @simon-friedberger the email is [info@homebase.id](mailto:info@homebase.id) Estimating 50,000 actual individual users per domain. 

@Seifert69 Thanks, please fix that in the PR.

> We've been working on Homebase since 2020, not sure what the required two years of validity is?

For validity see the [guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors).

**Comment by Seifert69** [comment]
@simon-friedberger The domains are now valid until sometime in 2027. I hope we fixed everything, otherwise please let us know. Appreciate your help & assistance 🙏🏼

**Comment by dnsguru** [comment]
I am unable to browse the org homepage https://www.homebase.id/ - is this operational?

**Comment by dnsguru** [comment]
I have been trying to access the homebase.id website (with and without the www) and only receive errors.  Is the project still active?  This is a bit of a red flag

I do notice that the project domain has an expiry date <2 years.

We have the 2 years rule in place to ensure that there is a certain amount of permanence to requested namespace(s) and their projects, as history has delivered a lot of disposable projects that made requests then had their org fail and nobody cleaned up after themselves, so the objective is to dump less future debris into the PSL and not disrespectfully burn through volunteer cycles in the process.

```
WHOIS / RDAP
homebase.id

Search Result 'homebase.id'
Domain ID	PANDI-DO7937104
Domain Name	homebase.id
Created	        2023-08-30 09:20:51
Last Updated	2023-10-04 11:32:48
Expired Date	2024-08-30 23:59:59  <--- Less than 1 year remaining!
Status              client transfer prohibited
```

**Comment by Seifert69** [comment]
Both fixed now. 

Sorry, screwed up the DNS earlier today and extended the PSL domains rather than the Hombase.id 🙄 😊

**Review by simon-friedberger — CHANGES_REQUESTED** [review]

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12691,6 +12691,12 @@ static.observableusercontent.com
 // Submitted by Andrew Sampson <andrew@ulterius.io>
 cya.gg
 
+// Homebase : https://homebase.id/
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12694]: After the namechange the sorting is wrong.

**Comment by Seifert69** [comment]
@simon-friedberger we believe the sorting is correct, but we're unsure if you (still) think the sorting is incorrect?

**Comment by simon-friedberger** [comment]
The sorting `one` first, then the `pub`s.

**Comment by Seifert69** [comment]
@simon-friedberger we already have one first, then pub?

*.id.pub
*.kin.one
*.kin.pub

In lines 11966 to 11968.

Sorry we're being so daft. We're trying really hard, honestly :-)

**Comment by OpieDelivers** [comment]
@simon-friedberger @Seifert69 I went ahead and changed this again, didn't realize it was sorted by tld first. Apologies!

**Review by simon-friedberger — APPROVED** [review]

**Comment by Seifert69** [comment]
Just checking politely in to see when this might get merged or if we're missing anything else? Best of holiday wishes.

**Review by dnsguru — APPROVED** [review]
* Sorted to Guidelines
* DNS TXT = Pull Request URL
* Tests Pass
* No conflict with base

APPROVED

**Pull request merged**